compiler/rustc_trait_selection/src/traits/project.rs

```diff
@@ -1180,7 +1171,7 @@ fn assemble_candidates_from_trait_def<'cx, 'tcx>(
         ProjectionCandidate::TraitDef,
         bounds.iter(),
         true,
-    )
+    );
```

Comment:
Is this diff new?

Reply:
function returns () but and the rest have semicolons, but i'll double check